### PR TITLE
[Cdn] Drop the definition suffix and rename Tls10 to Tls1_0

### LINKED ISSUE #26393: Release stable version Azure.ResourceManager.Cdn SDK

### PR DESCRIPTION
**Pull request by Yao725**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs [hunk 1]
```diff
@@ -336,7 +336,7 @@ public partial class AfdSecretData : Azure.ResourceManager.Models.ResourceData
     {
         public AfdSecretData() { }
         public Azure.ResourceManager.Cdn.Models.AfdDeploymentStatus? DeploymentStatus { get { throw null; } }
-        public Azure.ResourceManager.Cdn.Models.SecretDefinition Parameters { get { throw null; } set { } }
+        public Azure.ResourceManager.Cdn.Models.SecretProperties Parameters { get { throw null; } set { } }
```
Comment (ArcturusZhang) [inline sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs:339]: Should we also rename the property name to `Properties` to align the name of its type?

File: sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs [hunk 2]
```diff
@@ -373,7 +373,7 @@ public partial class AfdSecurityPolicyData : Azure.ResourceManager.Models.Resour
     {
         public AfdSecurityPolicyData() { }
         public Azure.ResourceManager.Cdn.Models.AfdDeploymentStatus? DeploymentStatus { get { throw null; } }
-        public Azure.ResourceManager.Cdn.Models.SecurityPolicyPropertiesDefinition Parameters { get { throw null; } set { } }
+        public Azure.ResourceManager.Cdn.Models.SecurityPolicyProperties Parameters { get { throw null; } set { } }
```
Comment (ArcturusZhang) [inline sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs:376]: similar question

File: sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs [hunk 3]
```diff
@@ -996,11 +996,11 @@ public partial class AfdRulePatch
     public partial class AfdSecurityPolicyPatch
     {
         public AfdSecurityPolicyPatch() { }
-        public Azure.ResourceManager.Cdn.Models.SecurityPolicyPropertiesDefinition Parameters { get { throw null; } set { } }
+        public Azure.ResourceManager.Cdn.Models.SecurityPolicyProperties Parameters { get { throw null; } set { } }
```
Comment (ArcturusZhang) [inline sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs:999]: similar question

**Review by ArcturusZhang — APPROVED** [review]

File: sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs (hunk 1 above)
Comment (Yao725) [reply sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs:339]: Nice suggestion! I will update.

**Pull request merged**